lib/oban.ex

```diff
@@ -647,6 +647,25 @@ defmodule Oban do
     |> Query.retry_job(job_id)
   end
 
+  @doc """
+  Retries all jobs that match on the given queryable. Please note that no matter the used
+  queryable it will never retry available, executing or scheduled jobs.
+  If no queryable is given, all jobs in states where retry is possible will be retried.
```

Comment:
```suggestion
  Retries all jobs that match on the given queryable. Please note that no matter the
  queryable constraints, it will never retry `available`, `executing` or `scheduled` jobs.
  
  If no queryable is given, Oban will retry all jobs in retryable states.
```